tap-snapshots/test/lib/commands/view.js.test.cjs

```diff
@@ -541,11 +541,7 @@ dist-tags:
 `
 
 exports[`test/lib/commands/view.js TAP workspaces remote package name > should have warning of ignoring workspaces 1`] = `
-Array [
-  Array [
-    "Ignoring workspaces for specified package(s)",
-  ],
-]
+Array []
```

Comment:
Need to investigate this one.

Reply:
Figured out this was due to the new log mocks having colors in them when `colors: true` is set in the config.

I changed the way the mocked logs are filtered to ignore color codes, but otherwise it will keep the logs colorized if the config is set.